liquibase-standard/src/test/java/liquibase/sqlgenerator/core/CreateTableGeneratorTest.java

```diff
@@ -1255,4 +1255,17 @@ public void testWithCreateIfNotExists() {
             }
```

Comment:
I think it would be nice if we could also add a test changeset into `https://github.com/liquibase/liquibase/blob/master/liquibase-integration-tests/src/test/resources/changelogs/oracle/complete/root.changelog.xml` with a create table change that include `rowDependencies` property. That way, at least we can make sure whether liquibase will process and deploy it successfully or not.

Reply:
Sure, i've added a changeset to the `root.changelog.xml` file for Oracle.